docs/_scripts/feature-selection.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+
+_file = Path(__file__)
+print(f"Executing {_file}")
+
+_static_path = Path("_static") / _file.stem
+_static_path.mkdir(parents=True, exist_ok=True)
+
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.datasets import fetch_openml
+from sklearn.ensemble import HistGradientBoostingClassifier
+from sklearn.feature_selection import f_classif, mutual_info_classif
+from sklearn.metrics import f1_score
+from sklearn.model_selection import train_test_split
+
+from sklego.feature_selection.mrmr import MaximumRelevanceMinimumRedundancy
+
+# --8<-- [start:mrmr]
+
+# Download MNIST dataset using scikit-learn
+mnist = fetch_openml("mnist_784", cache=True)
+
+# Assign features and labels
+X_pd, y_pd = mnist["data"], mnist["target"]
+
+X, y = X_pd.to_numpy(), y_pd.to_numpy()
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=10000, random_state=42)
+X_train = X_train.reshape(60000, 28 * 28)
+X_test = X_test.reshape(10000, 28 * 28)
+
+
+from scipy.spatial.distance import cosine
+
+
+def _redundancy_cosine_scipy(X, selected, left):
```

Comment:
Since it's public facing, maybe no underscore here?

```suggestion
def redundancy_cosine_scipy(X, selected, left):
```

Also would need to change below.

Reply:
I'm removing this anyway, I'll clean a bit the script removing unnecessary code.